tests/hpc/ww4.pm

```diff
@@ -53,6 +53,10 @@ sub run ($self) {
         record_info('authentication', 'container authentication is enabled');
     }
     my $hpc_container = get_required_var('HPC_WAREWULF_CONTAINER');
+
+    # Disable proxy-scc. See: progress.opensuse.org/issues/168028
```

Comment:
I would rather say:
`#  Disable url to use default SCC for repositories inside the warewulf-container`